tests/src/test/scala/cats/tests/SyntaxTests.scala

```diff
@@ -0,0 +1,181 @@
+package cats
+package tests
+
+import algebra.laws.GroupLaws
+import cats.functor.{Invariant, Contravariant}
+import cats.laws.discipline.SerializableTests
+
+import org.scalacheck.{Arbitrary}
+import org.scalatest.prop.PropertyChecks
+import scala.reflect.runtime.universe.TypeTag
+
+/**
+ * Test that our syntax implicits are working.
+ *
+ * Each method should correspond to one type class worth of syntax.
+ * Ideally, we should be testing every operator or method that we
+ * expect to add to generic parameters. This file is a safeguard
+ * against accidentally breaking (or removing) syntax which was
+ * otherwise untested.
+ *
+ * The strategy here is to create "mock" values of particular types,
+ * and then ensure that the syntax we want is available. We never plan
+ * to run any of these methods, so we don't need real values. All
+ * values in the methods should be generic -- we rely on parametricity
+ * to guarantee that the syntax will be available for any type with
+ * the proper type class instance(s).
+ *
+ * None of these tests should ever run, or do any runtime checks.
+ */
+class SyntaxTests extends CatsSuite with PropertyChecks {
+
+  // pretend we have a value of type A
+  def mock[A]: A = ???
```

Comment:
Not sure if this is well known or not, but I wonder if using `???` here might become a problem one day given: https://github.com/tpolecat/tpolecat.github.io/blame/master/_posts/2014-04-11-scalac-flags.md#L21